onset_maproom/calc.py

```diff
@@ -147,6 +252,26 @@ def cess_date(
 
 # Time functions
 def strftimeb2int(strftimeb):
+    """ Convert months as string values to integer values (1-12).
```

Comment:
I wrote this but I am not sure what is the best way to describe it due to my ad-hoc knowledge of Python. Is it just a Dictionary?

It might now matter as we are likely to evolve a great deal when it comes to time manipulation

Reply:
@drewmresnick here is the missing unresolved conversation.

In the latest version, it looks like you (or someone) removed the "(1-12)". I think that's important to keep, because in other places we represent months as 0-11.

Reply:
To address Rémi's comment, aside from what I just said I think this docstring is fine.